docs/workflows.rst

```diff
@@ -522,7 +524,18 @@ confounds per volume.
 Calculated confounds include the mean global signal, mean tissue class signal,
 tCompCor, aCompCor, Frame-wise Displacement, 6 motion parameters, DVARS, and, if
 the ``--use-aroma`` flag is enabled, the noise components identified by ICA-AROMA
-(those to be removed by the "aggressive" denoising strategy).
+(those to be removed by the "aggressive" denoising strategy). Note that
+ICA-AROMA uses FSL MELODIC for ICA decomposition, and FSL MELODIC estimates the
+number of independent components by means of a Bayesian model selection
+framework which assumes spherical Gaussian noise distribution. Although the
+model order estimation performs very well in general, it may overestimate the
+intrinsic dimensionality of highly autocorrelated datasets simply because these
+datasets fail to meet the model assumptions. What this means is that datasets
+with a very short TR and a large number of timepoints may end up with an
+unusually high number of components. For such particular use cases, fmriprep
+offers the ``--aroma-melodic-dimensionality`` option. This option allows users
+to enforce the number of components that MELODIC will extract (it is equivalent
+to the ``--dim`` option of the FSL MELODIC command line tool).
```

Comment:
Can you start every new sentence on a new line? It makes diffs cleaner if we make changes in the future by only showing the changed sentence in red/green.

Also, while this text is useful, it may be a little overwhelming for a section describing what columns are available in the confounds file. Perhaps something like the following?

```
The number of ICA-AROMA components depends on a dimensionality estimate
made by MELODIC.
For datasets with a very short TR and a large number of timepoints, this may
result in an unusually high number of components.
In such cases, it may be useful to specify the number of components to be
extracted with ``--aroma-melodic-dimensionality``.
```